com.ibm.wala.cast.java/build.gradle

```diff
@@ -0,0 +1,35 @@
+apply plugin: 'java'
+    
+repositories {
+    mavenCentral()
+        maven {
+    	    url "https://repo.eclipse.org/content/groups/releases/"
+	    }
+}
+
+
+
+sourceSets {
+    main {
+        java  {
+    	    srcDir 'src'
+    	    }
+    	}
+    test {
+        java {
+    	    srcDir 'test'
+    	    }
+    	}
+}
+
+
+dependencies {
+	compile project(':com.ibm.wala.cast')
+	compile project(':com.ibm.wala.core')
+	compile project(':com.ibm.wala.shrike')
+	compile project(':com.ibm.wala.util')
+	// https://mvnrepository.com/artifact/org.eclipse.core/org.eclipse.core.runtime
+	compile group: 'org.eclipse.core', name: 'org.eclipse.core.runtime', version: '3.10.0.v20140318-2214'
+	// https://mvnrepository.com/artifact/org.osgi/org.osgi.core
+	compile group: 'org.osgi', name: 'org.osgi.core', version: '4.2.0'
+}
```

Comment:
It would be nice to add the final newline here and in several of the other new `build.gradle` files.

Reply:
Yes, let's run all these files through some proper formatter (I think both Eclipse and IntelliJ can format `build.gradle` files)